index.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Mobile first Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"/>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header class="main-header">
+
+        <nav class="header">
+
+            <div class="name"><a href="#" style="color: white; text-decoration: none;">Usama Masood</a></div>
+            <div class="menu"><i class="fas fa-bars"></i></div>
+            
+        </nav>
+</header>
+    <main class="main1">
+        <section class="section1">
+            <div class="gap">gap</div>
+            <div class="intro">
+                <h1>Hey There. <br>
+                    I'm Usama
+                </h1>
+            </div>
+
+            <div class="profession"><h2>I'am a Software Developer</h2></div>
+
+            <div class="containe">
+                <div class="containe1">
+                    <div>
+                        <article class="services">
+                            <p>
+                                I can help you build a product , feature or <br>
+                                 website Look through some of my work <br>
+                                  and experience! If you like what you see <br>
+                                 and have a project you need coded, <br>
+                                don’t hestiate to contact me. <br>
+                            </p>
+                        </article>
+                        <ul class="logo">
+                            <li><a href="https://github.com/UsamaMasood12"><img src="./image/G.png" alt=""></a></li>
+                            <li><a href="www.linkedin.com/in/usama-masood-b4a35014b"><img src="./image/Group.png" alt=""></a></li>
+                            <li><a href="#"><img src="./image/M.png" alt=""></a></li>
+                            <li><a href="https://twitter.com/Usama____Masood"><img src="./image/twitter.png" alt=""></a></li>
+                            <li><a href="#"><img src="./image/V.png" alt=""></a></li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="containe2">
+                    <div class="box box1"><img src="./image/44.png" alt=""></div>
+                </div>
+            </div>
+          </section>
+        <section class="work">
```

Comment:
- [ ] [OPTIONAL] As stated in the project requirements kindly use CSS Grid in the main container that contains all of the cards in the work section 🙏